blocks/link-list/link-list.js

```diff
@@ -47,20 +68,32 @@ const buildLinkListItem = ({ textContent, url }) => {
  */
 
 export default async function decorate(block) {
-  const linkListContainer = document.createElement("div");
   const linkList = document.createElement("ul");
-  linkListContainer.classList.add("link-list");
-  linkListContainer.append(linkList);
+  linkList.classList.add("link-list");
 
-  const linkListLinksData = [...block.children].map((row) => ({
+  const linksData = [...block.children].slice(1).map((row) => ({
     textContent: [...row.children[0].children],
     url: row.children[1].innerText,
+    altText: row.children[2]?.innerText,
   }));
-
-  linkListLinksData.forEach((row) => {
+  if (linksData.every((link) => link.url.startsWith("https://adobe.design/careers/")))
+    linkList.classList.add("link-list--jobs");
+  linksData.forEach((row) => {
     const linkListItem = buildLinkListItem(row);
     linkList.append(linkListItem);
   });
 
-  block.replaceWith(linkListContainer);
+  block.replaceWith(linkList);
+
+  let footerLink;
+  let footerLinkData;
+  if (block.children[0].children[0].innerText && block.children[0].children[1].innerText) {
```

Comment:
I think we can clean this up a little bit. Go ahead and assign those values in your original data object with optional chaining. Then, if the values do not exist, they will populate as `undefined`. Your if statement here can then be something like `if footerLinkData.textContent && footerLinkData.url {}` which is much better for humans.

Reply:
Oh maybe we did this block before you really started to add thinks to links. Anywhere we can move those `block.children[0].whatever` chains to a named value before we start reassigning them or modifying them, we'll be doing our future devs a service.

Reply:
Don't know why I hadn't done this in the first place. Re-factored!